tests/component_handlers/schema_handler/test_schema_handler.py

```diff
@@ -149,8 +149,8 @@ def test_should_log_info_when_submit_schemas_that_not_exists_and_dry_run_true(
     )
     topic_config = TopicConfig(
         type=OutputTopicTypes.OUTPUT,
-        keySchema=None,
-        valueSchema="com.bakdata.kpops.test.SchemaHandlerTest",
+        key_schema=None,
```

Comment:
Is this breaking our existing configs?

Reply:
no, they use aliases for the camel case version